markdown/spring-cloud-stream-beyond/spring-cloud-stream-beyond.md

```diff
@@ -0,0 +1,476 @@
+
+![Solace Cloud Event Mesh Page](img/landing-page-event-mesh.webp "Solace Cloud Event Mesh")
+
+Click on 'Messaging Services' and all the messaging services associated with your account will show up if you have any already created. To create a new service, click either button as depicted in the image below:
+
+![Solace Cloud Landing Page](img/landing-page-signup.webp "Solace Cloud Landing Page")
+
+Fill out all the details for your messaging service, and then click "Create" at the bottom of the page. Note: make sure you choose the "Developer" option for the Service Type
+
+![Create Solace Cloud Messaging Service](img/create-service.png "Solace Cloud Messaging Service")
+
+Your service should be ready to use in a few minutes
+
+✅ (3) Take note of the connection details
+
+If your messaging service was created successfully, you'll be routed to the summary page of your new messaging service. From the service summary page, click on the "Connect" tab so we can take note of the connection details we'll need later.
+
+![Connect Tab Preview](img/service-summary-page.webp "Connect Tab Preview")
+
+After you click the "Connect" tab, click the "Connect with Spring" box, choose "Spring Cloud Stream", and click the "Connect to Service" option. We'll be using the "Secured SMF Host" URL in a minute 👍
+![Solace Web Messaging Tab](img/smf-messaging-connect-tab.webp "Solace Messaging Tab")
+
+### Create a Spring Cloud Stream Microservice 
+
+✅ (1) Create an Empty Spring Cloud Stream Microservice to use throughout this codelab 
+
+Navigate to [start.spring.io](https://start.spring.io/#!dependencies=solace,cloud-stream) and choose the "Solace PubSub+" and the "Cloud Stream" dependencies. If you used that link we already added them for you 😉.
+You can leave everything else as the default or modify the Java version, package info, etc. as desired. 
+
+Click the "Generate" button which will download the project. You can then unzip it and import the project into your preferred Java IDE as a maven project. I'll be using [Spring Tool Suite 4 for Eclipse](https://spring.io/tools) in the screenshots.
+Your imported project should look like this: 
+![Imported Project](img/importedProject.webp "Imported Project")
+
+✅ (2) Add your Connection Details from the Messaging Service
+
+Change the `application.properties` file to `application.yml` and copy and paste the `spring.cloud.stream.binders` part of the Spring Cloud Stream configuration previously found in the "Connect to Service" widget in your Message Service. 
+It should look something like this:
+
+``` yaml
+spring:
+  cloud:
+    stream:
+      binders:
+        solace-cloud:
+          type: solace
+          environment:
+            solace:
+              java:
+                clientUsername: default
+                clientPassword: default
+                host: 'tcp://localhost:55555'
+                msgVpn: default
+```
+
```

Comment:
Do you want to include a note along the lines of the host, username, pwd, and vpn are obtained from the previous step